recipes/_install-windows.rb

```diff
@@ -106,7 +106,7 @@
   # As of v1.37, the windows cookbook doesn't upgrade the package if a newer version is downloaded
   # As a workaround uninstall the package first if a new MSI is downloaded
   code <<-EOH
-((New-Object System.Net.WebClient).DownloadFile('https://s3.amazonaws.com/ddagent-windows-stable/scripts/fix_6_14.ps1', $env:temp + '\\fix_6_14.ps1')); &$env:temp\\fix_6_14.ps1
+((New-Object System.Net.WebClient).DownloadFile('#{node['datadog']['windows_agent_url']}scripts/fix_6_14.ps1', $env:temp + '\\fix_6_14.ps1')); &$env:temp\\fix_6_14.ps1
```

Comment:
should we maybe use a helper var and do:
```
URI::join(node['datadog']['windows_agent_uri'], 'scripts/fix_6_14.ps1') 
```
?

Reply:
OK, just realized we don't do this for the `remote_file` source, so we wouldn't even get this far. Probably not something we need to do.

Reply:
indeed I used this for consistency, I think we can update both to use `URI::join` in a later PR